metrics/network/remote_network/remote-networking-iperf3.sh

```diff
@@ -149,8 +157,6 @@ function remote_network_latency {
 }
 
 function main {
-	[[ $# -ne 7 ]]&& help && die "Illegal number of parameters."
```

Comment:
Did you mean to sneak a couple of other changes in here?
If so, can you break them out into a separate commit pls?